agenta-cli/agenta/cli/main.py

```diff
@@ -106,10 +111,10 @@ def init(app_name: str):
             if global_backend_host:
                 backend_host = global_backend_host
             else:
-                backend_host = "https://cloud.agenta.ai"
+                backend_host = "http://localhost"
```

Comment:
why?

Reply:
Ooops, I forgot to change it back after development.